cypress/integration/examples/edit_product.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+
+import Chance from "chance";
+const chance = new Chance();
+
+const randomProduct = chance.integer({ min: 0, max: 48 });
+const randomCheckbox = chance.integer({ min: 0, max: 6 });
+let newRandomCheckbox;
+if (randomCheckbox === 0) {
+  newRandomCheckbox = 1;
+} else if (randomCheckbox === 6) {
+  newRandomCheckbox = 0;
+} else {
+  newRandomCheckbox = chance.integer({ min: 2, max: 5 });
+}
```

Comment:
You generate a random number between 0 and 6 (`randomCheckbox`).
If this number is `0`, you set `newRandomCheckbox` to `1`. 
If it is `6`, you set `newRandomCheckbox` to `0`. 
If it is 2,3,4 or 5, you generate a new integer between 2 and 5. 

Thus, it's possible that `newRandomCheckbox` is 0, 1, 2, 3, 4 or 5. 
This sounds too complex 😵.
Why not call `chance.integer({ min: 0, max: 5 });` in the first place?